docs/components/concepts/signals.md

```diff
@@ -0,0 +1,19 @@
+---
+id: signals
+title: "Signals"
+description: "Broadcast signals to trigger signal events"
+---
+
+Signals can be broadcasted within Zeebe. These broadcasts are caught by [signal catch events](../modeler/bpmn/signal-events/signal-events.md).
+Once a signal is caught the process instance will respond accordingly.
```

Comment:
🔧 Let's start by introducing what signals are: e.g. Signals are similar to Messages. But they are also different because they trigger all matching signal events, where messages are correlated to a specific process instance.

🔧 Let's also avoid referring to Camunda Platform 8 instead of Zeebe, because Signals also exist in Web and Desktop Modeler, Operate, and Optimize.